pkg/interfaces/contracts/pool-hooks/IMevHook.sol

```diff
@@ -56,6 +69,26 @@ interface IMevHook {
     /// @notice Permissioned function to enable charging the MEV Tax in registered pools.
     function enableMevTax() external;
 
+    /**
+     * @notice Returns maximum MEV swap fee percentage returned by `onComputeDynamicSwapFeePercentage`.
+     * @dev The absolute minimum is still the static swap fee percentage of the pool. If the maximum MEV swap fee
+     * percentage of the pool is below the static fee percentage, the static fee percentage is used anyways.
```

Comment:
```suggestion
     * @dev The absolute minimum is still the static swap fee percentage of the pool.
```
Given the explanation right below, the second sentence is redundant. (And "anyways" is slang; should be "anyway" if used.)